tox.ini

```diff
@@ -353,11 +338,11 @@ passenv = *
 extras = all
 deps =
```

Comment:
This whole env needs to be deduplicated; hence why the above recommended split into the three envs, then hopefully, the diff to multi and base is clear